pretix_telephone/views.py

```diff
@@ -0,0 +1,34 @@
+from django import forms
+from django.urls import reverse
+from django.utils.translation import ugettext_lazy as _
+
+from pretix.base.forms import SettingsForm
+from pretix.base.models import Event
+from pretix.control.views.event import EventSettingsFormView, EventSettingsViewMixin
+
+
+class TelephoneFieldSettingsForm(SettingsForm):
+    telephone_field_required = forms.BooleanField(
+        label=_("Require phone number"),
+        help_text=_("If this is not checked, entering a phone number is optional.."),
+        required=False,
+    )
+    telephone_field_help_text = forms.CharField(
```

Comment:
This should be an I18nField to allow for multilingual help texts.

Reply:
Wow...this happening to me... will fix xD